File: 백준/Bronze/10214. Baseball/Baseball.py
import sys

t = int(sys.stdin.readline())
yonsei, korea = 0, 0
for _ in range(t):
    yonsei, korea = 0, 0
    for _ in range(9):
        y, k = map(int, sys.stdin.readline().split())
        yonsei += y
        korea += k
    if yonsei < korea:
        print('Korea')
    elif korea < yonsei:
        print('Yonsei')
    else:
        print('Draw')

File: 백준/Bronze/10214. Baseball/Baseball.cc
#include <iostream>
using namespace std;

int main() {
	cin.tie(0)->sync_with_stdio(0);

	int t;
	cin >> t;
	while (t--) {
		int yonsei = 0, korea = 0;
		for (int i = 0; i < 9; i++) {
			int y, k;
			cin >> y >> k;
			yonsei += y;
			korea += k;
		}
		if (yonsei < korea) cout << "Korea" << '\n';
		else if (yonsei > korea) cout << "Yonsei" << '\n';
		else cout << "Draw" << '\n';
	}
}